common/src/main/java/io/netty/util/internal/logging/JdkLoggerFactory.java

```diff
@@ -25,6 +25,15 @@
  */
 public class JdkLoggerFactory extends InternalLoggerFactory {
 
+    public static final InternalLoggerFactory INSTANCE = new JdkLoggerFactory();
+
+    /**
+     * @deprecated Use {@link #INSTANCE} instead.
+     */
+    @Deprecated
+    public JdkLoggerFactory() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)